judgments/tests/test_detail.py

```diff
@@ -424,6 +427,44 @@ def test_breadcrumb_when_judgment(self, mock_judgment, mock_get_pdf_size):
         </div>"""
         assert_contains_html(response, breadcrumb_html)
 
+    @patch("judgments.views.detail.get_pdf_size")
+    @patch("judgments.views.detail.get_judgment_by_uri")
+    @pytest.mark.parametrize(
+        "http_error,expected_breadcrumb",
+        [
+            (JudgmentNotFoundError, "Page not found"),
+            (Exception, "Server Error"),
```

Comment:
havent added test for 403 Forbidden even though we do have a template for that.

We dont have any pages that are authenticatd so we should get rid of that.

On the other hand we should be adding a 400 Bad request page for cases detailed in https://trello.com/c/k5JDtcKL/1129-dont-show-500-on-invalid-uris-show-400-bad-request-in-pui